docs/src/css/custom.css

```
@@ -3707,3 +3707,283 @@ a.connect-card {
+/*    B["Database"]:::success                                            */
+/*    class B ic-database            (icon + a role color together)      */
+/*  The icon inherits the label color (white on a role block, else ink). */
+/* -------------------------------------------------------------------- */
+.docusaurus-mermaid-container .node:is(.ic-app,.ic-user,.ic-service,.ic-datastore,.ic-key,.ic-lock,.ic-globe,.ic-shield,.ic-flow,.ic-token,.ic-check,.ic-alert) .nodeLabel::before {
+  content: "";
+  display: inline-block;
+  width: 1.05em;
+  height: 1.05em;
+  margin-right: 0.45em;
+  vertical-align: -0.18em;
+  background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)